src/app/app.component.ts

```diff
@@ -64,13 +66,23 @@ export class AppComponent implements OnInit {
   public url = 'https://twitter.com/AngularClass';
 
   constructor(
-    public appState: AppState
+    public appState: AppState,
+    public appEnv: AppEnv
   ) {}
 
   public ngOnInit() {
     console.log('Initial App State', this.appState.state);
+    this.initGoogleAnalytics();
```

Comment:
this is easier to manage but it's better for these types of services to live outside so they're setup asap

Reply:
I dont know do I understand you. Can you explain a little bit more what do you mean when you talk about 'these types of services" and "live outside"? Do you speak about 'initGoogleAnalytics' or 'AppEnv'? What exactly do you expect?  Please be concrete and clear - this will allow me to make proper fix.

Reply:
He means the starter meant to be at the basic level, as this line that he comments on is referring to another service that might not be needed as a "starter" if I'm starting project, the Google Analytics should be added in the last phase of my project not as a starting point that will waste sometime just to "start"! in short words, starter should be startup and run without pre-configuration.

Reply:
Im not 100% sure that I understand you @almothafar - hoverer currently in code (on master - not in this pull request) googleAnalytics is invoked "at the top" of html file (so even before angular starts):
https://github.com/AngularClass/angular-starter/blob/master/src/index.html

The goal of this pull request was not changing (or fixing) this initGA behaviour but introduce support for .env files. I think better place to fix this separate topic will be another pull request - however I can change it here if you really want but tell me exactly where in code you wan to initialize GA (give fileName-line coordinates) because I not think about this problem and Im not sure - ?

Reply:
@kamil-kielczewski the solution is really simple, just comment out this line and keep it, so who wants it he can remove the comment mark and enable it, the code in index.html is not going to break anything.

And yes indeed you introducing support for .env files so no need to this line at all, it belongs to another pull request or feature.

I prefer to delete this line as what I said in lastly.

Reply:
ok, if @gdi2290 approve your proposition I will do it (because in fact comment-out change pre-pull behaviour).